src/sql/src/parsers/copy_parser.rs

```diff
@@ -39,10 +39,73 @@ impl<'a> ParserContext<'a> {
                     actual: self.peek_token_as_string(),
                 })?;
 
-        self.parser
-            .expect_keyword(Keyword::TO)
+        if self.parser.expect_keyword(Keyword::TO).is_ok() {
```

Comment:
Could we use `parse_keyword()`?
```suggestion
        if self.parser.parse_keyword(Keyword::TO) {
```